core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -317,15 +317,15 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
 
     // Validate testUser counts
-    val result = runAndWait(workflowQuery.countWorkflowsByQueueStatusByUser)
+    val result: Map[String, Map[String, Int]] = runAndWait(workflowQuery.countWorkflowsByQueueStatusByUser)
 //    result should equal(Set())
     result should contain key (testUserId)
     testUserStatusCounts.foreach { case (st, count) =>
       result(testUserId)(st.toString) should be (count)
     }
 
     // Validate all workspace counts by status
-    validateCountsByQueueStatus(result.combineAll)
+    validateCountsByQueueStatus(Monoid.combineAll(result.values))
```

Comment:
Both [rawls](https://github.com/broadinstitute/rawls/blob/develop/project/Dependencies.scala#L58) and [wdl4s](https://github.com/broadinstitute/wdl4s/pull/255/files#diff-0ecdbc5a001d52fb34f5eafb7cd1aaa6R4) directly specify cats 0.9.0, but cats 1.0.0-MF shows up and evicts a bunch (but not all) of the sub-libraries of 0.9.0, seemingly more in `an-GAWB-2641` than in `develop`.

Unfortunately I don't have a good explanation for why cats 1.0.0-MF shows up, but its presence potentially explains the need to make this code change - there are a number of [breaking changes](https://github.com/typelevel/cats/blob/master/CHANGES.md#to-migrate-from-090) when upgrading from 0.9.0

Reply:
wdl4s 0.15-hash depends on `async-http-client-backend-cats:0.0.16`, which depends on `cats-effect_2.11:0.4`, which depends on `cats-core_2.11:1.0.0-MF` and `cats-laws_2.11:1.0.0-MF`

* https://github.com/broadinstitute/wdl4s/blob/0.15_hotfix/project/Dependencies.scala#L39
* https://search.maven.org/remotecontent?filepath=com/softwaremill/sttp/async-http-client-backend-cats_2.11/0.0.16/async-http-client-backend-cats_2.11-0.0.16.pom
* https://search.maven.org/remotecontent?filepath=org/typelevel/cats-effect_2.11/0.4/cats-effect_2.11-0.4.pom

Reply:
Very cool. Did you figure this out from the wdl4s `dependencyTree`? I searched through it but there were hundreds of references to cats `1.0.0-MF`, virtually all of which were just noting that `0.9.0` had been evicted

Reply:
yup, `sbt dependency-graph > /some/file`, then a bunch of manual hunt and peck in BBEdit.